pylint/testutils/output_line.py

```diff
@@ -80,7 +80,7 @@ def from_msg(cls, msg):
 
     @classmethod
     def get_column(cls, column):
-        if sys.version_info.major == 3 and sys.version_info.minor < 8:
+        if PY38_PLUS:
```

Comment:
I think there might be a bug here...

Reply:
You're right, it should be `not PY38_PLUS` :D